.github/workflows/lint-test.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     - name: Check code formatting with Black
       run: |
        black --version
-       black --check .
+       black --check --diff .
```

Comment:
OOC: What is the reasoning for changing this line?

Reply:
https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#:~:text=echo%20%24%3F%0A123-,%2D%2Ddiff,-%C2%B6

When we push changes, it isnt helpful to just know that black failed, we want to see the diff that black would apply to fix the issue